scheduler/scheduler.go

```diff
@@ -96,7 +96,8 @@ func (s *scheduler) processInstance(ctx context.Context, si *scheduledInstance)
 					log.Printf("Error running task %s on instance %s. Got: %v\n", task.Name(), si.instance.Name, err)
 					// Since one task failed, we just assume something might be wrong with the instance, so we don't try to process the rest of the tasks.
 					si.fails++
-					if si.fails > 5 {
+					// As each task runs 1/sec then keep trying up to 5 minutes
+					if si.fails > 300 {
```

Comment:
Maybe we should move this into a flag and leave 300 as the default?

Reply:
ok, I'll move it.